Test/test_cmor_license_template.py

```diff
@@ -0,0 +1,161 @@
+import json
+import cmor
+import unittest
+import os
+import numpy
+
+from netCDF4 import Dataset
+
+LICENSE_INFO = {
+    "license_template":
+        ("CMIP7 data produced by <institution_id> is licensed under a <license_type> "
+        "License (<license_url>). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse "
+        "for terms of use governing CMIP7 output, including citation requirements "
+        "and proper acknowledgment. The data producers and data providers make no "
+        "warranty, either express or implied, including, but not limited to, "
+        "warranties of merchantability and fitness for a particular purpose. All "
+        "liabilities arising from the supply of the information (including any "
+        "liability arising in negligence) are excluded to the fullest extent "
+        "permitted by law."),
+    "license_id":{
+        "CC BY 4.0":{
+            "license_type":"Creative Commons Attribution 4.0 International",
+            "license_url":"https://creativecommons.org/licenses/by/4.0/"
+        },
+        "CC0 1.0":{
+            "license_type":"Creative Commons CC0 1.0 Universal Public Domain Dedication",
+            "license_url":"https://creativecommons.org/publicdomain/zero/1.0/"
+        }
+    }
+}
+
+DATASET_INFO = {
+    "_AXIS_ENTRY_FILE": "Tables/CMIP6_coordinate.json",
+    "_FORMULA_VAR_FILE": "Tables/CMIP6_formula_terms.json",
+    "_cmip7_option": 1,
+    "_controlled_vocabulary_file": "TestTables/CMIP7_CV.json",
```

Comment:
The `TestTables/CMIP7_CV.json` file will likely need to be updated to match the `LICENSE_INFO` format above

Reply:
Will CMIP7 use the new license method?

I'll need to update the CMIP7 test, which might make the test in this PR redundant.

Reply:
We have the CV Task Team on this thread @taylor13 @ltroussellier @znichollscr @sashakames @wolfiex, so am keen to hear from them. I think this implementation is a good idea indeed.